x/staking/client/cli/tx.go

```diff
@@ -66,8 +66,11 @@ func NewCreateValidatorCmd() *cobra.Command {
 				return err
 			}
 
-			txf := tx.NewFactoryCLI(clientCtx, cmd.Flags()).
-				WithTxConfig(clientCtx.TxConfig).WithAccountRetriever(clientCtx.AccountRetriever)
+			txf, err := tx.NewFactoryCLI(clientCtx, cmd.Flags())
```

Comment:
is there any reason to omit the `TxConfig`, `AccountRetriever` here?

Reply:
Bc it's redundant code. It's already set in `NewFactoryCLI`